chainlink-testing-framework/run-tests/action.yml

```diff
@@ -116,6 +116,12 @@ runs:
       shell: bash
       run: ${{ inputs.build_gauntlet_command }}
 
+    # gotestfmt gives us pretty test output
+    - name: Set Up gotestfmt
+      uses: GoTestTools/gotestfmt-action@v2
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }} # Avoids rate-limiting
```

Comment:
The token will need to be an input since an action doesn't have access to the secrets.